### PR TITLE
Update attestation control plane SDK to new version

### PR DESCRIPTION
**Pull request by shleiAmy**
Link to the seagger spec review PR:
https://github.com/Azure/azure-rest-api-specs/pull/7490

update attestation control plane SDK

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @shleiAmy , 
- could you add a link to the swagger spec review PR in the description
- for feature addition, consider updating the minor version, aka `0.10.0`
- please also update `AssemblyFileVersion` in `AssemblyInfo.cs`

Thanks

**Comment by shleiAmy** [comment]
@isra-fel swagfger spec review PR link is added.
   and versions are updated.

File: sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -36,4 +36,4 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
```
Comment (isra-fel) [inline sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyVersion("0.10.0.0")]
```

File: sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -36,4 +36,4 @@
...
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.1.0.0")]
```
Comment (isra-fel) [inline sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyFileVersion("0.10.0.0")]
```

File: sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (isra-fel) [reply sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs]: The general rule for preview module is: update assemblyversioon and assemblyfileversion to be the same as the version in csproj. Once it goes stable, update assemblyfileversion only unless it's a major version update.

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (shleiAmy) [reply sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs]: update to 
[assembly: AssemblyVersion("0.10.0.0")]
[assembly: AssemblyFileVersion("0.10.0.0")]

File: sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (shleiAmy) [reply sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs]: @isra-fel 
update to
[assembly: AssemblyVersion("0.10.0.0")]
[assembly: AssemblyFileVersion("0.10.0.0")]

**Review by isra-fel — APPROVED** [review]

**Pull request merged**